src/CMakeLists.txt

```diff
@@ -54,6 +52,9 @@ ELSE()
             ${CMAKE_DL_LIBS})
 ENDIF()
 
+# Add compile definitions
+TARGET_COMPILE_DEFINITIONS(${KHIVALIB} PRIVATE KHIVA_STATICLIB PUBLIC AF_UNIFIED)
```

Comment:
does this definition clash with the one in line 22?

Reply:
@raulbocanegra it does not clash, but line 22 is no longer needed. Please remove it and we are done to merge